src/natten/functional.py

```diff
@@ -1191,7 +1196,8 @@ def forward(
         ctx.tiling_config_backward = tiling_config_backward
         ctx.has_bias = bias is not None
 
-        return output
+        return output, logsumexp, maximums
```

Comment:
Let's definitely not return either of `logsumexp` or `maximums` when `return_statistics` is `False`.

Reply:
Actually nevermind, that doesn't make sense. I need to think more about how we're returning three things here.